### PR TITLE
Bepro 1557 my netwok page new management tab design

### PR DESCRIPTION
**Pull request by MarcusviniciusLsantos**

### CODE REVIEW

File: components/list-issues.tsx [hunk 1]
```diff
@@ -92,9 +93,13 @@ export default function ListIssues({
 
   const [searchState, setSearchState] = useState(search);
 
+  const isManagement = variant === 'management';
   const isProfile = variant === "profile";
   const isBountyHall = variant === "bounty-hall";
   const isOnNetwork = !!router?.query?.network;
+  const variantIssueItem = isManagement ? variant : (isProfile || isBountyHall) ? "multi-network" : "network"
+  const columns = ["Name", "Bounty Link", "Hide Bounty", "Cancel"];
```
Comment (vhcsilva) [inline components/list-issues.tsx:101]: Missing translation

File: components/profile/my-network-settings/index.tsx [hunk 2]
```diff
@@ -291,6 +293,15 @@ export default function MyNetworkSettings({
           </NetworkContainer>
         ),
       },
+      {
+        eventKey: "management",
+        title: "Management",
```
Comment (vhcsilva) [inline components/profile/my-network-settings/index.tsx:298]: Missing translation

File: pages/api/network/management/index.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    if (!issue) return res.status(404).json({message: "Issue not found"});
+
+    issue.visible = visible
+
+    await issue.save()
+
+
+    return resJsonMessage("Bounty updated", res, 200);
+  } catch (error) {
+    console.log(error);
```
Comment (vhcsilva) [inline pages/api/network/management/index.ts:71]: use `Logger` service

**Review by vhcsilva — CHANGES_REQUESTED** [review]
I think that instead of passing `visible: true` every time we make a request, we could set `visible: true` as the default on all endpoints. In this way we guarantee that if we forget to pass this parameter, no hidden bounties will be returned.

**Review by vhcsilva — APPROVED** [review]

**Review by moshmage — APPROVED** [review]

**Pull request merged**